src/Symfony/Component/Console/Command/Command.php

```diff
@@ -447,6 +448,26 @@ public function getName()
     }
 
     /**
+     * @param bool $public Whether the command should be publicly shown or not.
+     * 
+     * @return Command The current instance
+     */
+    public function setPublic($public)
```

Comment:
What about `setPrivate()` without an argument? I don't see a need to switch from private to public.